windows/logdna-agent.yml

```diff
@@ -0,0 +1,26 @@
+http:
+  host: logs.logdna.com
+  endpoint: /logs/agent
+  use_ssl: true
+  timeout: 10000
+  use_compression: true
+  gzip_level: 2
+  params:
+    hostname: Dmitri-PC
```

Comment:
is this correct?

Reply:
windows folder is wip, should go to separate MSI commit.